README.md

```diff
@@ -8,157 +8,155 @@
-  Unleash server.
+Unleash Edge is a fast and lightweight proxy layer between your Unleash API and SDKs. It acts as a read replica of your Unleash instance and is designed to help you scale Unleash. It allows you to support thousands of connected SDKs without increasing the number of requests you make to your Unleash instance.
 
-Unleash Edge is built to help you scale Unleash.
+Edge supports both client-side and server-side SDKs and has multi-environment and project awareness. You can daisy-chain Edge instances to support more complex setups, such as multi-cloud deployments.
 
-* If you're looking for the easiest way to connect your client SDKs you can check out
-  our [Frontend API](https://docs.getunleash.io/reference/front-end-api).
-* If you're looking to learn how to scale your own feature flag system why not check out our recommendations for
-  building and scaling [feature flags](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+Key features:
 
-## Migrating to Edge from the Proxy
+- **Performance**: Edge uses in-memory caching and can run close to your end-users. A single instance can handle tens to hundreds of thousands of requests per second.
+- **Resilience**: Edge is designed to survive restarts and maintain functionality even if you lose connection to your Unleash server.
+- **Security**: Edge supports frontend applications without exposing sensitive data to end-users or to Unleash.
 
-For more info on migrating, check out the [migration guide](./migration-guide.md) that details the differences between
-Edge and the Proxy and how to achieve similar behavior in Edge.
+You can run Edge in two different modes: **edge** or **offline**. To learn about the different modes and other Edge concepts, visit [Concepts](./docs/concepts.md).
+
+Unleash Edge is the successor to Unleash Proxy. For help with migrating from Proxy to Edge, refer to the [migration guide](./migration-guide.md).
+  
+If you're looking for the simplest way to connect your client SDKs, explore our [Frontend API](https://docs.getunleash.io/reference/front-end-api). For additional recommendations on scaling your feature flag system, see our [Best practices for building and scaling feature flags](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices) guide.
 
 ## Quickstart
 
-Running Edge in Docker with our recommended setup:
+Our recommended approach is to bootstrap Edge with a client API token and upstream URL as command line arguments or container environment variables.
+
+To run Edge in Docker:
 
 ```shell
-docker run -it -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<yourunleashinstance> -e TOKENS=<yourclienttoken> unleashorg/unleash-edge:<mostrecentversion> edge
+docker run -it -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<your_unleash_instance> -e TOKENS=<your_client_token> unleashorg/unleash-edge:<mostrecentversion> edge
 ```
 
 ## Edge behaviors
```

Comment:
Love the rewrite here @melindafekete - so much clearer. Thank you for helping out :)